pkg/codegen/internal/test/testdata/simple-enum-schema/python/pulumi_plant_provider/tree/v1/rubber_tree.py

```diff
@@ -18,7 +19,7 @@ def __init__(__self__,
                  resource_name: str,
                  opts: Optional[pulumi.ResourceOptions] = None,
                  container: Optional[pulumi.Input[pulumi.InputType['_root_inputs.ContainerArgs']]] = None,
-                 type: Optional[pulumi.Input[enums.RubberTreeVariety]] = None,
+                 type: Optional[pulumi.Input[_enums._RubberTreeVariety]] = None,
```

Comment:
One thing I've been wondering: The test schema only has `inputProperties` for the `RubberTree` resource. What do output properties look like for enums?

 - If it's a strict enum, will it be typed as the enum type? (And therefore needs to be converted to the enum type during deserialization?)

- If it's a relaxed enum (e.g. `Union[EnumClass, str]`), rather than the output being typed as `Union[EnumClass, str]`, should it be just `str` to make it easier for consumers (i.e. it will always be a string)?